src/containers/Landing.js

```diff
@@ -1,43 +1,54 @@
 import React, { Component } from "react";
 import { Link } from "react-router-dom";
+import AppLogo from '../images/AppLogo.svg';
+import AdaLogo from '../images/AdaLogo.png';
 
 class Landing extends Component {
   render() {
     return (
       <div>
-        <header class="main__header">
-            <a class="header__link" href="index.html">
-            <img class="header__image" src="assets/images/Awesome-profile-cards.svg" alt="Awesome profile-cards logo"></img>
+        <header className="main__header">
```

Comment:
This page does not contain a header so the code between the header tags (is included) should not be

Reply:
Good point @valerialp! I just removed it, should be fine now.